<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminUI</class>
 <widget class="QMainWindow" name="adminUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Libreria</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#adminUI{
	background-color:#193E38
}

QLineEdit{
	background-color:white;
}
QLabel{
	color:black;
}
QPushButton{
    background-color: #4CAF50;
    border: none;
    color: white;
    padding: 5px 15px;
}
QPushButton:pressed{
    background-color:#006400;
    border: none;
    color: white;
    padding: 5px 15px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QTabWidget" name="maintab">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="books">
       <attribute name="title">
        <string>LIBRI IN LIBRERIA</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Cerca:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>NOME LIBRO:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="searchLine"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>AUTORE:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="autoreLine"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>ARGOMENTO:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="genereLine"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="tableView"/>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Libro:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0" rowspan="2">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="2">
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="coverLabel">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="xml.qrc">:/img/nocopertina.png</pixmap>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="labelIS">
                 <property name="text">
                  <string>ISBN:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelName">
                 <property name="text">
                  <string>Titolo:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labeldate">
                 <property name="text">
                  <string>Data pub.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelA">
                 <property name="text">
                  <string>Autore/i:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelCP">
                 <property name="text">
                  <string>Copie disponibili:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelDesc">
                 <property name="text">
                  <string>Descrizione:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="2">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLineEdit" name="lineIS">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineName">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineDate">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineA">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineCP">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineDesc">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Azioni:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="getBookButton">
             <property name="text">
              <string>CHIEDI PRESTITO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addnewB">
             <property name="text">
              <string>AGGIUNGI LIBRO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="text">
              <string>CANCELLA LIBRO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>AGGIORNA DATABASE</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="userbooks">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <attribute name="title">
        <string>MIEI LIBRI</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>Libro:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="0" column="0" rowspan="2">
              <layout class="QGridLayout" name="gridLayout_13">
               <item row="0" column="2">
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0">
                <spacer name="horizontalSpacer_9">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="coverLabelU">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="xml.qrc">:/img/nocopertina.png</pixmap>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <spacer name="horizontalSpacer_10">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="labelIS_3">
                 <property name="text">
                  <string>ISBN:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelName_3">
                 <property name="text">
                  <string>Titolo:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labeldate_3">
                 <property name="text">
                  <string>Data pub.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelA_3">
                 <property name="text">
                  <string>Autore:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelDesc_3">
                 <property name="text">
                  <string>Descrizione:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="2">
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="QLineEdit" name="lineISU">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineNameU">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineDateU">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineAU">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineDescU">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Azioni:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPushButton" name="returnButton">
             <property name="text">
              <string>RESTITUISCI LIBRO</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>Libri in mio possesso:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QTableView" name="tableviewuser"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>UTENTI</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>Cerca:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>NOME UTENTE:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="sLineID"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_22">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>NOME:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="sLineName"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_23">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>COGNOME:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="sLineSname"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Utenti:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableView" name="tableUsersView"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>ID utente:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineIDU">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addUser">
            <property name="text">
             <string>AGGIUNGI UTENTE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeUser">
            <property name="text">
             <string>TOGLI UTENTE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccout">
    <property name="title">
     <string>Accout</string>
    </property>
    <addaction name="actionAccount_informations"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuAccout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAccount_informations">
   <property name="text">
    <string>Account informations</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="xml.qrc"/>
 </resources>
 <connections/>
</ui>
